config/i18n.js

```diff
@@ -1,5 +1,13 @@
-export default {
+const clientI18n = {
   langPickerRender: true,
   langList: ['en-US', 'ru-RU'],
   fallbackLng: 'en-US'
 };
+
+const serverI18n = {
+  locales: ['en', 'ru'],
+  directory: process.cwd() + '/src/modules/i18n/locales',
```

Comment:
You are adding `directory` config parameter because you have one directory for translations for all the backend. This is wrong. The client-side i18n were made modular, and all the kit is modular. You should have per-module translations as was done for client i18n. It should be easy to remove module from the kit - you just delete directory of the module and remove import of that module. You shouldn't break this mechanism.